Mathlib/Data/Rel.lean

```diff
@@ -168,6 +193,21 @@ theorem image_univ : r.image Set.univ = r.codom := by
   simp [mem_image, codom]
 #align rel.image_univ Rel.image_univ
 
+@[simp]
+theorem image_empty : r.image ∅ = ∅ := by
+  ext x
+  simp [mem_image]
+
+@[simp]
+theorem image_bot (s : Set α) : (⊥ : Rel α β).image s = ∅ := by
+  rw[Set.eq_empty_iff_forall_not_mem]
+  intro x h
+  simp[mem_image, Bot.bot] at h
+
+theorem image_top {s : Set α} (h : Set.Nonempty s) :
```

Comment:
This definitely deserves a `@[simp]` (since `simp` can often figure out whether `Set.Nonempty` is true).
```suggestion
@[simp]
theorem image_top {s : Set α} (h : Set.Nonempty s) :
```